project/Dependencies.scala

```diff
@@ -21,7 +21,7 @@ object Dependencies {
   // https://github.com/real-logic/aeron/blob/1.x.y/build.gradle
   val agronaVersion = "1.8.0"
   val nettyVersion = "3.10.6.Final"
-  val jacksonVersion = "2.10.5"
+  val jacksonVersion = "2.10.5.1"
```

Comment:
```suggestion
  val jacksonVersion = "2.10.5"
  val jacksonDatabindVersion = "2.10.5.1"
```

then, where jackson-databind is listed we need to use the new version. It is a recurring situation in the Jackson ecosystem that jackson-databind needs these hotfixe versions that miss-align with the rest of the artifacts. I think keeping this new value will make it more maintainable especially since it's scala-steward doing the hard work.